sonar-python-plugin/src/main/java/org/sonar/plugins/python/caching/Caching.java

```diff
@@ -81,9 +87,10 @@ public void writeProjectLevelSymbolTableEntry(String moduleFqn, Set<Descriptor>
     cacheContext.getWriteCache().write(cacheKey, toProtobufModuleDescriptor(descriptors).toByteArray());
   }
 
-  public void copyFromPrevious(String moduleFqn) {
-    cacheContext.getWriteCache().copyFromPrevious(IMPORTS_MAP_CACHE_KEY_PREFIX + moduleFqn);
-    cacheContext.getWriteCache().copyFromPrevious(PROJECT_SYMBOL_TABLE_CACHE_KEY_PREFIX + moduleFqn);
+  public void copyFromPrevious(String fileKey) {
+    cacheContext.getWriteCache().copyFromPrevious(IMPORTS_MAP_CACHE_KEY_PREFIX + fileKey);
+    cacheContext.getWriteCache().copyFromPrevious(PROJECT_SYMBOL_TABLE_CACHE_KEY_PREFIX + fileKey);
+    cacheContext.getWriteCache().copyFromPrevious(CONTENT_HASHES_KEY + fileKey);
```

Comment:
If you change the cache key according to the suggestion above, do not also forget to update here.